tools/jenkins/slaves/Dockerfile.centos

```diff
@@ -40,7 +42,7 @@ RUN scl enable rh-ror50 $HOME/bushslicer/tools/hack_bundle.rb
 
 RUN rpm -qa && \
     yum clean all -y && \
-    rm -rf $HOME/bushslicer /var/cache/yum /tmp/*
+    /var/cache/yum /tmp/*
```

Comment:
where did `rm -rf` go? I think it makes more sense though to remove tierN and private here instead of earlier. Put things together should be fine IMO.